clients/python-wrapper/lakefs/branch.py

```diff
@@ -236,12 +238,18 @@ def revert(self, reference_id: str, parent_number: Optional[int] = None) -> None
         elif parent_number <= 0:
             raise ValueError("parent_number must be a positive integer")
 
+        # Handle reference_id as a deprecated alias to reference.
+        reference = reference or reference_id
```

Comment:
1. Please issue warning when `reference_id` is not None
2. Please validate `reference` is not None afterwards